### DISCUSSION TITLE
what is turn server

### DISCUSSION THREAD

**Discussion by atiqueansar** [Mupati/laravel-video-chat#6] [General]
TURN_SERVER_URL=
TURN_SERVER_USERNAME=
TURN_SERVER_CREDENTIAL=

**Comment by amitjoshi2188** [comment]
I have integrated the whole repo, but am stuck on TURN_SERVER-related details, can you guide how to manage those details?
or any other repo, which can do video calling without TURN_SERVER-related details.

**Comment by ravirajladha** [comment]
Yeah, same here. i am unable to run the webrtc video call on localhost. without the clarification of this TURN credentials, how to achieve that.
